src/ReactiveDomain.Debug.nuspec

```diff
@@ -3,28 +3,32 @@
   <metadata>
     <id>ReactiveDomain</id>
     <version>0.10.5.0</version>
-    <authors>PerkinElmer,Linedata</authors>
+    <authors>Reactive Domain Group</authors>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <license type="expression">MIT</license>
     <description>Package includes all ReactiveDomain Core assemblies</description>
     <dependencies>
-      <group targetFramework="net48">
+      <group targetFramework="net8.0">
         <dependency id="EventStore.Client" version="21.2.2" exclude="Build,Analyzers" />
         <dependency id="Microsoft.CSharp" version="4.7.0" exclude="Build,Analyzers" />
-        <dependency id="Newtonsoft.Json" version="13.0.2" exclude="Build,Analyzers" />
-        <dependency id="System.Configuration.ConfigurationManager" version="7.0.0" exclude="Build,Analyzers" />
-        <dependency id="System.Diagnostics.PerformanceCounter" version="5.0.1" exclude="Build,Analyzers" />
+        <dependency id="Newtonsoft.Json" version="13.0.3" exclude="Build,Analyzers" />
+        <dependency id="ReactiveUI" version="20.1.1" exclude="Build,Analyzers"/>
```

Comment:
Don't forget to update the dependencies in the nuspec files

Reply:
?? I did those

Reply:
oh, you mean dropping reactiveUI not updating the json version